diff -uNr freeminer-0.4.9.3/src/json/CMakeLists.txt freeminer-0.4.9.3-1/src/json/CMakeLists.txt
--- freeminer-0.4.9.3/src/json/CMakeLists.txt	2014-01-22 21:36:09.000000000 +0400
+++ freeminer-0.4.9.3-1/src/json/CMakeLists.txt	2014-08-19 12:56:47.106387980 +0400
@@ -6,7 +6,8 @@
 	set(json_platform_LIBS "")
 endif( UNIX )
 
-add_library(jsoncpp ${json_SRCS})
+#add_library(jsoncpp ${json_SRCS})
+add_library(jsoncpp STATIC ${json_SRCS})
 
 target_link_libraries(
 	jsoncpp
